src/app/backend/auth/api/types.go

```diff
@@ -16,6 +16,11 @@ package api
 
 import "k8s.io/client-go/tools/clientcmd/api"
 
+const (
+	EncryptionKeyHolderName      = "kubernetes-dashboard-key-holder"
+	EncryptionKeyHolderNamespace = "kube-system"
```

Comment:
It doesn't really matter as long as dashboard will have access to this secret. It can access it from any namespace. Our yaml files will contain correct permissions. If user decides to change them then it's his responsibility to make sure they are valid.

I'd only think how to configure multiple dashboard deployments to use different secrets (not replicas of single deployment). We might add some argument to dashboard as suffix that will be used for secret creation. Autogeneration and sync between multiple replicas might be hard.